src/components/index.ts

```diff
@@ -1 +1,4 @@
-export { default as Navbar } from "./Navbar/Navbar";
+export { default as Navbar } from "./Navbar";
+export { default as Modal } from "./Modal";
+export { default as InstructionsComponent } from "./InstructionsComponent";
+export { default as QuizCard } from "./QuizCard";
```

Comment:
Include `CategoryCard` as well here, I think you missed that component

Reply:
done